### ISSUE TITLE
Flameshot randomly screenshots entire screen even if cropped

### ISSUE THREAD

**Issue by non3mu** [flameshot-org/flameshot#1278]
**Flameshot version**
v0.8.5

**System information**
Linux Mint 20.1 Cinnamon (xorg)

**Expected behavior**
Cropping a screenshot will actually crop it

**Actual behavior**
Flameshot randomly copies the whole screen instead of just the cropped area

**Comment by mmahmoudian** [comment]
Can you record your screen and show in action what happens. You can use `peek` to record gif or mp4

https://github.com/phw/peek

**Comment by sonulohani** [comment]
I am using Linux mint 20.1 and I am not able to reproduce this issue.

**Comment by non3mu** [comment]
It's hard to reproduce because it only seems to happen randomly but I'll try to record it next time it happens. I've had a friend have the same issue on Manjaro

**Comment by mmahmoudian** [comment]
I have manjaro on two sepa machines and cannot reproduce it.

**Comment by gabrc52** [comment]
I can confirm this bug and just reproduced it some seconds ago. Where can I get the log file?

**Comment by mmahmoudian** [comment]
@gabrc52 at the moment the closest thing to a log we have is to:
1. open two terminals
2. In the first terminal type
```sh
pkill flameshot
flameshot
```
3. In the second terminal type
```sh
flameshot gui
```
4. Do the whatever you have to do to reproduce the bug and copy or save to file
5. The first terminal should show you some errors if it has faced any

**Comment by tilderain** [comment]
I think I've found a way to reproduce it. It happens when you drag-click to make a selection box, but press enter to take the screenshot before letting go of the mouse button. I guess the selection only takes effect only after mouse up.

**Comment by mmahmoudian** [comment]
Yes, few months ago @veracioux  changed the code in a way to only accept the input keys after the mouse release. For this reason no keyboard shortcut works while you are dragging. That was done to handle similar PEBCEK error as user can be in hurry and do things in wrong order. If @non3mu can also confirm this was the source of the issue, we can close this thread.

**Comment by mmahmoudian** [comment]
Perhaps #4498 have addressed this issue. If you can, try compiling from the master branch and let us know is this issue still persists. I'm going to close this for now, but if the issue persists, we will re-open it.
